package.json

```diff
@@ -37,11 +37,11 @@
     "prettier": "^2.5.1",
     "semantic-release": "^19.0.2",
     "ts-jest": "^27.1.3",
-    "typescript": "^4.5.5",
-    "uuid": "^8.3.2"
+    "typescript": "^4.5.5"
   },
   "dependencies": {
-    "@types/aws-lambda": "^8.10.92"
+    "@types/aws-lambda": "^8.10.92",
+    "uuid": "^8.3.2"
```

Comment:
This was already in use in `dynamo-streams.ts` -- it should have been in `dependencies` already